packages/analytics/amplify_analytics_pinpoint/pigeons/pigeon_config_android.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// To regenerate, run `make pigeons`.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(
+  PigeonOptions(
+    dartOut:
+        'lib/src/legacy_native_data_provider/pigeon_data_provider.android.g.dart',
+    javaOut:
+        'android/src/main/java/com/amazonaws/amplify/amplify_analytics_pinpoint/amplify_analytics_pinpoint/Messages.java',
+    javaOptions: JavaOptions(
+      className: 'Messages',
+      package:
+          'com.amazonaws.amplify.amplify_analytics_pinpoint.amplify_analytics_pinpoint',
+    ),
+    copyrightHeader: 'pigeons/copyright.txt',
+  ),
+)
+@HostApi()
+abstract class PigeonDataProvider {
```

Comment:
Can we name this more descriptively? e.g. something like `LegacyDataProvider`.

Reply:
This was marked as resolved but I think still applies

Reply:
My bad, I thought I had renamed it but was clearly mistaken.  Thanks for catching!